src/actions/rating.js

```diff
@@ -0,0 +1,18 @@
+import { ARTICLE_RATING_ERROR } from './types';
+import axios from '../utils/axiosInstance';
+import authUtils from '../utils/auth';
+
+const postArticleRating = (ratingValue, slug) => async (dispatch) => {
+  try {
+    const userToken = authUtils.getUserToken();
+
+    axios.defaults.headers.common.Authorization = userToken;
+
+    await axios.post(`/articles/${slug}/rating`, ratingValue);
+  } catch (response) {
```

Comment:
@marcdomain this catch block here, are you using the variable name `response` to catch the error? 
Another thing, if the you're saying `const errorMessage = response` on line 13 below, will `errorMessage` be a string or an object?

Reply:
Good observation. 
The catch block is actually meant to catch errors for users that are not logged in and trying to rate the article. For now the catch error is not dispatched (it's redundant) as the `isLogedIn` in our `store` is been fixed in Luqman's PR. So for now, if you're not logged in, you won't get an error message, even though your rating would not be submitted to the API (you only see the frontend animation that gives you a feel that you've rated the article). The dispatch in the catch block would be fixed as Luqman's PR is merged.
This implementation actually has no issues as it is, but would be made more elegant briefly.

Reply:
ok then 👍